etc/kayobe/inventory/host_vars/ethsw-b16-u38

```diff
@@ -146,6 +146,11 @@ switch_interface_config:
       - "no shutdown"
       # Controllers.
       - "tagged Te 1/5/2,1/5/4"
+  vlan 41:
+    description: overcloud provisioning
+    config:
+      - "no shutdown"
+      - "untagged Te 1/4/4"
```

Comment:
Why is this required?

Reply:
This is a relic from when we were going to move the nodes into bifrost. This was before we realised there was no physical connection between the 1G and 10G networks. We should remove this.